celeryutils/persist_on_failure.py

```diff
@@ -0,0 +1,51 @@
+"""
+Celery utility code for persistent tasks.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from celery import Task
+
+from .models import FailedTask
+
+
+# pylint: disable=abstract-method
```

Comment:
There is an abstract method defined on `celery.Task` (`run()`), that isn't overridden here, because this is also an abstract task base class, but celery uses its own annotation for abstract classes, so pylint doesn't understand it.